src/Models/ArduinoDeviceBase.ts

```diff
@@ -160,20 +160,46 @@ export abstract class ArduinoDeviceBase implements Device {
   abstract get version(): string;
 
   private async writeCppPropertiesFile(
-      rootPath: string, boardId: string, type: ScaffoldType,
-      cppPropertiesTemplateFileName: string,
-      cppPropertiesFilePath: string): Promise<void> {
+      boardId: string, type: ScaffoldType, plat: string): Promise<void> {

```

Comment:
It is better to use full name as platform for easy to understand

Reply:
Will fix.